### PR TITLE
Fix/many tooltips

### PR DESCRIPTION
**Pull request by alexgoncharwix**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_393/

File: src/components/ColorPicker/ColorPicker.spec.tsx [hunk 1]
```diff
@@ -8,9 +8,28 @@ import { ColorPicker } from './';
...
+    // this data-hook is taken from the wix-ui-core/Popover implementation:
+    // https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/popover/Popover.uni.driver.ts#L17
+    // TODO fix TooltipDriver in core, to remove the portal when not needed
+    const portal =
+      document && document.querySelector('[data-hook="popover-portal"]');
+
+    if (portal) {
+      portal.remove();
+    }
+  });
```
Comment (jonathanadler) [inline src/components/ColorPicker/ColorPicker.spec.tsx:32]: So after the fix I made to the tooltips and the driver, this part isn't needed

File: src/components/ColorPicker/ColorPicker.spec.tsx [hunk 2]
```diff
@@ -82,7 +101,34 @@ describe('ColorPicker', () => {
...
+
+    const itemDriverFirst = driver.getItemAt(0);
+    const itemDriverSecond = driver.getItemAt(1);
+
+    expect(await itemDriverFirst.getTooltipText(bodyUniDriver)).toBe(
+      'ArrowTop.svgRed',
+    );
+    expect(await itemDriverSecond.getTooltipText(bodyUniDriver)).toBe(
+      'ArrowTop.svgBlue',
+    );
```
Comment (jonathanadler) [inline src/components/ColorPicker/ColorPicker.spec.tsx:131]: This should now work out-of-the-box

File: src/components/ColorPicker/ColorPickerItem/ColorPickerItem.spec.tsx [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+    // this data-hook is taken from the wix-ui-core/Popover implementation:
+    // https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/popover/Popover.uni.driver.ts#L17
+    // TODO fix TooltipDriver in core, to remove the portal when not needed
+    const portal =
+      document && document.querySelector('[data-hook="popover-portal"]');
+
+    if (portal) {
+      portal.remove();
+    }
+  });
```
Comment (jonathanadler) [inline src/components/ColorPicker/ColorPickerItem/ColorPickerItem.spec.tsx:26]: again, this can be removed

**Review by jonathanadler — COMMENTED** [review]
@alexgoncharwix 
Do you want to "refresh" this PR?

**Review by jonathanadler — APPROVED** [review]

**Review by jonathanadler — APPROVED** [review]

**Pull request merged**